library/runlevel/test/service_test.rb

```diff
@@ -35,7 +35,7 @@ def stub_service_with(method, result)
       end
 
       it "returns the result of the original result of the command call" do
-        expect(Service.call("status", "sshd")).to be_kind_of(String)
+        expect(Service.call("status", "sshd")).to be_kind_of(::String)
```

Comment:
Ouch, we should get rid of the `String` module, the conflict with the standard Ruby class makes me always crazy...